internal/common/metrics/grpc.go

```diff
@@ -63,16 +63,17 @@ func NewGRPCInterceptor(defaultHandler Handler, suffix string) grpc.UnaryClientI
 
 		// Only take method name after the last slash
 		operation := method[strings.LastIndex(method, "/")+1:]
-		tags := map[string]string{OperationTagName: operation}
 
 		// Since this interceptor can be used for clients of different name, we
 		// attempt to extract the namespace out of the request. All namespace-based
 		// requests have been confirmed to have a top-level namespace field.
+		namespace := "_unknown_"
```

Comment:
I wonder if an empty string is better. Is there a common industry approach for label placeholder values?

Reply:
I don't know if there is a standard. I might have picked a different value myself, but this is what the server uses for the same situation so it seemed like a good idea to match